app/assets/javascripts/discourse/app/routes/user-private-messages-group.js

```diff
@@ -1,4 +1,21 @@
-import createPMRoute from "discourse/routes/build-private-messages-group-route";
-import { INBOX_FILTER } from "discourse/routes/build-private-messages-route";
+import DiscourseRoute from "discourse/routes/discourse";
 
-export default createPMRoute("group", INBOX_FILTER);
+export default class extends DiscourseRoute {
+  model(params) {
+    return params.name;
+  }
+
+  setupController(controller, model) {
+    controller.set("groupName", model);
+  }
+
+  renderTemplate() {
```

Comment:
I came across this comment by @davidtaylorhq in https://github.com/discourse/discourse/pull/19224#issuecomment-1329593503 so we might need another pattern here.

Reply:
This has been resolved. I created a component which uses the `{{in-element}}` modifier which allows a child route to render into an element in the parent route. This solution was suggested by the Ember's deprecation doc.